R/label.R

```diff
@@ -1,18 +1,40 @@
 #' Assign Variable Label
 #'
 #' Assigns variable label from a variable level metadata to a given data frame.
+#' This function will give an error if a label is presented that is greater than
+#' 40 characters which isn't allowed in XPT v5. If labels aren't present for the
+#' variable it will be assigned an empty character value. Labels are stored in
+#' the 'label' attribute of the column.
+#'
+#' @inheritParams xportr_length
+#'
+#' @section Messaging: `label_log` is the primary messaging tool for
+#'   `xportr_label`. If there are any columns present in the '.df' that are not
+#'   noted in the metadata, they cannot be assigned a label and a message will
+#'   be generated noting the number or variables that have not been assigned a
+#'   label.
+#'
+#'   If variables were not found in the metadata and the value passed to the
+#'   'verbose' argument is 'stop', 'warn', or 'message', a message will be
+#'   generated detailing the variables that were missing in metadata.
+#'
+#' @section Metadata: The argument passed in the 'metadata' argument can either
+#'   be a metacore object, or a data.frame containing the data listed below. If
+#'   metacore is used, no changes to options are required.
+#'
+#'   For data.frame 'metadata' arguments three columns must be present:
+#'
+#'   1) Domain Name - passed as the 'xportr.domain_name' option. Default:
+#'   "dataset". This is the column subset by the 'domain' argument in the
+#'   function. 2) Variable Name - passed as the 'xportr.variable_name' option.
+#'   Default: "variable". This is used to match columns in '.df' argument and
+#'   the metadata. 3) Variable Label - passed as the 'xportr.label' option.
```

Comment:
Formatting is a bit wonky here as well 
![image](https://github.com/atorus-research/xportr/assets/10111024/2b7e78ad-7a54-480c-86bf-30ba7b761b94)

Reply:
Fixed